manifests/init.pp

```diff
@@ -254,6 +258,12 @@
     }
   }
 
+  if $pulp3 and $pulp_master {
+    foreman::config::apache::fragment { 'pulp':
```

Comment:
For now, you could put the pulpcore deinfition here as well:

```
class { 'pulpcore':
  manage_apache: false
}
```

And add it as a dependency

Reply:
@ehelms done, thanks

Reply:
This fragment might should be named `pulpcore` given we already have one named `pulp` and they will clash:

```
[ERROR 2019-12-06T02:12:44 verbose]  Evaluation Error: Error while evaluating a Resource Statement, Duplicate declaration: Foreman::Config::Apache::Fragment[pulp] is already declared at (file: /usr/share/foreman-installer/modules/katello/manifests/pulp.pp, line: 39); cannot redeclare (file: /usr/share/foreman-inst
aller/modules/foreman_proxy_content/manifests/init.pp, line: 265) (file: /usr/share/foreman-installer/modules/foreman_proxy_content/manifests/init.pp, line: 265, column: 5)
```

Reply:
done, thanks